graphene_django/filter/fields.py

```diff
@@ -89,11 +90,22 @@ def connection_resolver(
         **args
     ):
         filter_kwargs = {k: v for k, v in args.items() if k in filtering_args}
-        qs = filterset_class(
+
+        filterset = filterset_class(
             data=filter_kwargs,
             queryset=default_manager.get_queryset(),
             request=info.context,
-        ).qs
+        )
+
+        if not (filterset.is_bound and filterset.form.is_valid()):
+            exc = {
+                str(key): [str(e.message) for e in error_list]
```

Comment:
The key here is in Python/Django source casing I believe. I think we could improve the error reporting here anyway (see other comment), but if we _do_ want to send these to the client, they should use the field names as in the API – i.e. probably `camelCase` names, although I believe this can be configured per schema, and overridden per field.